packages/cerebral/src/Module.js

```diff
@@ -14,8 +15,12 @@ class Module {
     module.signals = Object.keys(module.signals || {}).reduce((currentSignals, signalKey) => {
       const signal = module.signals[signalKey]
 
-      currentSignals[signalKey] = (payload) => {
-        controller.runSignal(path.concat(signalKey).join('.'), signal, payload)
+      currentSignals[signalKey] = {
+        signal: signal.signal || signal,
+        catch: signal.catch || controller.catch,
```

Comment:
Fixed... merging `Map` is a little bit complicated, must explicitly spread it to merge it in, in case you wonder why I solved it that way. Anyways, fixed and test added.